src/commands/createFunction/durableSteps/netherite/EventHubsNamespaceCreateStep.ts

```diff
@@ -22,8 +23,9 @@ export class EventHubsNamespaceCreateStep<T extends IEventHubsConnectionWizardCo
         progress.report({ message: creating });
 
         const client: EventHubManagementClient = await createEventHubClient(<T & ISubscriptionContext>context);
+        const location: AzExtLocation = await LocationListStep.getLocation(<T & ISubscriptionContext>context);
```

Comment:
Does this work? I wasn't sure if you had to `LocationListStep.setLocation` first, after getting it from the `context.resourceGroup.location`.

Reply:
Yeah it works, I looked at `ResourceGroupListStep` and found that it actually sets it for you:

![image](https://user-images.githubusercontent.com/40250218/205185522-2953bd7b-fd9d-48c7-a5ef-7f3fd6b4c751.png)

Reply:
I believe it works because the location is set with `StorageAccountListStep`.  Let's talk offline about this, there's some questions I want to address now that I'm looking into the Azure Wizard Step utils more closely.

Reply:
Discussed:  Sounds like we can probably keep this `getLocation` call and investigate whether or not it makes sense to add a `setLocation` in `ResourceGroupListStep` as a part of `Azure Tools`.